utilities/pegin-pegout-automation-demo/README.md

```diff
@@ -0,0 +1,110 @@
+# Flyover SDK Pegin/Pegout Automation Demo
+
+This utility provides automated demonstration flows for testing and showcasing the Flyover SDK's pegin and pegout automation functionalities. It includes interactive demos with clear step-by-step presentations and error handling scenarios.
+
+## Overview
+
+The automation demo includes several flows designed to demonstrate different aspects of the Flyover bridge:
+
+- **Pegin Flows**: Move Bitcoin to RSK (RBTC)
+- **Pegout Flows**: Move RBTC to Bitcoin
+- **Testing Scenarios**: Error handling, capacity limits, and signature validation
+
+## Prerequisites
+
+- Having the local environment running with the flyover sdk installed and configured.
+- Having a valid mnemonic and addresses for testing in local.
```

Comment:
I'd clarify the mnemonic needs to be the one capable of derivating the whitelisted address